easybuild/easyconfigs/n/nodejs/nodejs-16.15.1-GCCcore-11.3.0.eb

```diff
@@ -14,10 +14,18 @@ toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
 
 source_urls = ['https://nodejs.org/dist/v%(version)s/']
 sources = ['node-v%(version)s.tar.gz']
-checksums = ['308aee7149c4092a53c87c28ef49e23a8d1606119e79ae68333062e2a1f94208']
+patches = [
+    '%(name)s-%(version)s_fix-argument-list-too-long.patch',
+]
+checksums = [
+    {'node-v16.15.1.tar.gz': '308aee7149c4092a53c87c28ef49e23a8d1606119e79ae68333062e2a1f94208'},
+    {'nodejs-16.15.1_fix-argument-list-too-long.patch':
+     '36b5627e60fd2b861ce3dc5445b137b4fe7ec30a4f866ea540ec56bda9686405'},
+]
 
 builddependencies = [
     ('binutils', '2.38'),
+    ('pkg-config', '0.29.2'),
```

Comment:
`pkg-config` is no longer maintained, so from `2022a` onward we prefer to use `pkgconf` where possible